### ISSUE TITLE
Namespace and Form issue

### ISSUE THREAD

**Issue by AngularPlayer** [angular-translate/angular-translate#1763]
When i use a form under  a translate-namespace directive i couldn't access my form from $scope in my controller, But he is happening in $scope.$$childHead instead. this is not a very welcome behavior. is it a normal behavior ?

version of angular-translate : 2.15.2
version of angular : 1.6.4

### HTML
```
<div translate-namespace=".myComponentName">
     <form name="myFormName" action="" novalidate>
                <input type="email" required/>
     </form>
</div>
```

### JS
```
function controller($scope){
      if($scope.$$childHead.myFormName.$valid){
      //do someting
      }
}
```

**Comment by knalli** [comment]
Hello,

as we had already mentioned in the [issue's template](https://github.com/angular-translate/angular-translate/blob/master/.github/ISSUE_TEMPLATE.md), we need a working and as minimal as possible demo covering your specific use case or issue. 

Follow these steps:
1. Please open [this plnkr base](https://plnkr.co/edit/8vRLnfGmtg51qZFfDeVc?p=preview).
2. Fork it (at the top left).
3. Ensure the version of AngularJS and angular-translate is correct.
4. Ensure all angular-translate plugins are available; reduce and remove anything you can.
5. Write a minimal as possible demo for your specific issue.
6. Save and freeze. Provide us the final link to your demo.

**Comment by AngularPlayer** [comment]
Hi, here the [plnkr](https://plnkr.co/edit/XwTAMRSJqiQgTwx2i9E7?p=preview) : 
You can see the controller of myForm switch from $scope to $scope.$$childHead and inversely by adding or removing the translate-namespace directive.

**Comment by knalli** [comment]
Personally, I would recommend the last option, because I would always try not using `$scope` indirectly but a controller scope whenever it is possible. Well, yes, `app` (controller scope) is `$scope.app` actually, but I hope you got the point. That's much more like Angular2 apps would be, and ensure data access is always being "with a dot".

**Comment by AngularPlayer** [comment]
I agree, i didn't know we are able to do that, your last suggestion solution is just perfect :+1:
